frontend/src/components/dashboard/Dashboard.vue

```diff
@@ -1,35 +1,26 @@
 <script setup lang="ts">
-import { computed, onMounted, onUnmounted, ref, reactive } from 'vue';
 import router from '@/router';
 import Dropdown, { type DropdownChangeEvent } from 'primevue/dropdown';
-import ConfirmDialog from 'primevue/confirmdialog';
-import { useConfirm } from 'primevue/useconfirm';
+import { computed, onMounted, onUnmounted, ref } from 'vue';
 
+import NotificationMessage from '@/components/common/NotificationMessage.vue';
 import DashboardTitle from './DashboardTitle.vue';
 import UserDataTable from './UserDataTable.vue';
-import NotificationMessage from '@/components/common/NotificationMessage.vue';
 
 import { ApiServiceFactory } from '@/services/ApiServiceFactory';
 import {
-    applicationsUserAdministers,
-    isApplicationSelected,
-    selectedApplication,
-    setSelectedApplication,
-    selectedApplicationDisplayText,
+applicationsUserAdministers,
+isApplicationSelected,
+selectedApplication,
```

Comment:
maybe we need the space for better formatting?

Reply:
Ah~ interesting, that's due to auto formatting from vscode.
I don't think I have setup the format you guys have....
Thanks!